generatehtml.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+
+import json
+import os
+
+import codecoverage
+
+
+def generate_html(json_file_folder):
```

Comment:
Let's call this `data_folder` instead, since we're storing many things there and not just a json file.